src/main/webapp/itempagescript.js

```diff
@@ -138,28 +136,33 @@ function deleteComment(commentId) {
       });
 }
 
-
 /**
  * Creates list element from given comment
  *
- * @param { string } comment - the comment including the username, message, and
- *     timestamp
- * @param { boolean } belongsToUser - boolean that shows whether or not the
- *     current user has posted a comment
- * @param { string } email - the email of the user who posted the comment
- * @returns { html element }  returns a list element
+ * @param { JSON } comment - the comment whose data is displayed on the list
+ *     element
+ * @returns { jQuery }  returns a list element
  */
-function createListElement(comment, belongsToUser, email) {
+function createListElement(comment) {
   const liElement = $('<li class="list-group-item"></li>');
-  liElement.text(comment);
-  if (belongsToUser) {
+  const date = new Date(comment.timestampMillis);
+
+  liElement.text(
+      comment.username + ': ' + comment.comment + ' - ' +
+      '(' + date.toLocaleString() + ')');
+
+  liElement.append(
+      $('<img class="pr-1" src="' + getProfileImageUrl(comment.email) +
+        '.png?s=23' +
+        '"style=float:left;"></img>'));
+
```

Comment:
super tiny nit: can remove extra line

Reply:
For some reason the formatter is adding this extra line